### DISCUSSION TITLE
I don't understand how to apply to GitHub Education as a student

### DISCUSSION THREAD

**Discussion by DiogoMiguelFonsecaSantos** [community/community#141844] [GitHub Education]
### Hi everyone,

I'm trying to apply for GitHub Education since I'm a student of the Higher Institute of Engineering of Lisbon (ISEL) and when I'm in the page "Upload proof of your academic status", it is not only asked of me to upload qualifying proof of current academic status but also "an image supporting your reason for being far from campus". I do have not only my university card with and ID and a photo of myself but also a registration certificate, proof that I'm studying there. My first question would be: why can't i upload an image where the information is easily identifiable instead of taking a really low-quality  picture of one of the two things with a laptop webcam/ phone webcam that does not take quality pictures? The other question and the one that im really struggling with is: I'm in campus. Since I'm in campus, why would i need to upload an image supporting your reason for being far from campus? Do i just ignore the second upload request? I've tried countless times to send the image with the proof of my academic status, ignoring the second request but it never works and I can't talk directly to GitHub support since this seems to not be a reason to approach them (there's no option to select in the topic of the ticket). I hope i made myself clear while writting this since english is not my first language. I really needed Copilot on VSCode since I'm struggling with programming subjects in university. If any of you answer this, i'll be more than grateful.

**Comment by ZionPat** [comment]
I have the same exact issue.
